DependencyInjection/DoctrineExtension.php

```diff
@@ -143,9 +144,18 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
         $container->setParameter('doctrine.connections', $connections);
         $container->setParameter('doctrine.default_connection', $this->defaultConnection);
 
+        /** @psalm-suppress UndefinedClass */
+        if (interface_exists(Middleware::class)) {
+            $container
+                ->getDefinition('doctrine.dbal.logger')
+                ->replaceArgument(0, null);
```

Comment:
This disables logging even when it's enabled, right? Basically it will block release of 2.6 once we merge this, until you finish follow-up PR

Reply:
It disables logging but the logging is now done by the logging middleware, see `Resources/config/middlewares.xml` in this PR.

Reply:
that's just abstract middleware

Reply:
Yes but abstract middlewares are registered for each connection by `MiddlewarePass` 🙂